gravitee-apim-console-webui/src/management/api/creation-v4/steps/step-5-summary/step-5-summary.component.html

```diff
@@ -16,166 +16,168 @@
+                </div>
+                <div class="step-5-summary__step__info__security__content__card__content">
+                  <span>Entrypoints type: http</span>
+                </div>
               </div>
             </div>
+            <ng-template #elseBlock> No plans are selected. </ng-template>
+          </div>
+          <div>
+            <button mat-stroked-button (click)="onChangeStepInfo('Security')">Change</button>
           </div>
-          <ng-template #elseBlock> No plans are selected. </ng-template>
-        </div>
-        <div>
-          <button mat-stroked-button (click)="onChangeStepInfo('Security')">Change</button>
         </div>
       </div>
+      <div *ngIf="shouldUpgrade">
+        <gio-license-banner
+          [license]="license$ | async"
+          [isOEM]="isOEM$ | async"
+          (onRequestUpgrade)="onRequestUpgrade()"
+        ></gio-license-banner>
+      </div>
     </div>
-    <div *ngIf="shouldUpgrade">
-      <gio-license-banner
-        [license]="license$ | async"
-        [isOEM]="isOEM$ | async"
-        (onRequestUpgrade)="onRequestUpgrade()"
-      ></gio-license-banner>
-    </div>
-  </div>
-  <div class="step-5-summary__footer">
-    <div class="step-5-summary__footer__actions">
-      <button mat-stroked-button type="button" (click)="createApi({ deploy: false, askForReview: false })">
-        Save API<mat-icon
-          svgIcon="gio:info"
-          class="step-5-summary__footer__button-icon"
-          matTooltip="Your API definition will be saved, but not deployed to the gateway."
-        ></mat-icon>
-      </button>
-      <button
-        *ngIf="!hasReviewEnabled"
-        data-testid="deploy_api_button"
-        color="primary"
-        mat-flat-button
-        type="button"
-        [disabled]="shouldUpgrade"
-        (click)="createApi({ deploy: true, askForReview: false })"
-      >
-        Save & Deploy API<mat-icon
-          svgIcon="gio:info"
-          class="step-5-summary__footer__button-icon"
-          [matTooltip]="
-            deployable
-              ? 'Your API will be deployed to the gateway.'
-              : 'You cannot deploy an API with endpoints or entrypoints not deployed.'
-          "
-        ></mat-icon>
-      </button>
-      <button
-        *ngIf="hasReviewEnabled"
-        data-testid="create_and_ask_review_api_button"
-        color="primary"
-        mat-flat-button
-        type="button"
-        (click)="createApi({ deploy: false, askForReview: true })"
-      >
-        Save API & Ask for a review<mat-icon
-          svgIcon="gio:info"
-          class="step-5-summary__footer__button-icon"
-          matTooltip="Your API definition will be saved and asked for review."
-        ></mat-icon>
-      </button>
+    <div class="step-5-summary__footer">
+      <div class="step-5-summary__footer__actions">
+        <button mat-stroked-button type="button" (click)="createApi({ deploy: false, askForReview: false })">
```

Comment:
The design is broken here we miss some margin on the icon I think.

<img width="354" alt="Capture d’écran 2024-02-15 à 16 32 37" src="https://github.com/gravitee-io/gravitee-api-management/assets/25704259/f054a975-274c-4061-84bc-c63bd01df957">

Reply:
good catch. 
Like all component override, the best thing to do is 🚮 the css